app/scss/adminPage.scss

```diff
@@ -104,15 +103,15 @@ input {
   }
 
   .download-user-results-button {
-    background-color: $primary-button;
+    background-color: $delete-user-button;
```

Comment:
If this is the download user button you might want to change the name of this variable